h5io/_h5io.py

```diff
@@ -211,6 +231,8 @@ def _triage_read(node):
         if type_str == 'dict':
             data = dict()
             for key, subnode in node.items():
+                for key_spec, val_spec in special_chars.items():
```

Comment:
ok - I didn't put that in there just to keep things simple but I can add that in